src/Avalonia.Svg/AvaloniaModelExtensions.cs

```diff
@@ -117,61 +117,50 @@ public static AM.TextAlignment ToTextAlignment(this SKTextAlign textAlign)
-
-            case SKFontStyleWeight.ExtraBlack:
-                return AM.FontWeight.ExtraBlack;
-        }
-    }
-
-    public static AM.FontStyle ToFontStyle(this SKFontStyleSlant fontStyleSlant)
-    {
-        switch (fontStyleSlant)
-        {
-            default:
-            case SKFontStyleSlant.Upright:
-                // TODO: FontStyleSlant.Upright
-                return AM.FontStyle.Normal;
-            case SKFontStyleSlant.Italic:
-                return AM.FontStyle.Italic;
-
-            case SKFontStyleSlant.Oblique:
-                return AM.FontStyle.Oblique;
-        }
-    }
+    public static AM.FontWeight ToFontWeight(this SKFontStyleWeight fontStyleWeight) =>
+        // The numeric values in these two enumerations have the same scale.
+        // Casting is even encouraged in AM.FontWeight XML documentation.
+        (AM.FontWeight)fontStyleWeight;
+    public static SKFontStyleWeight ToSKFontWeight(this AM.FontWeight fontStyleWeight) =>
+        (SKFontStyleWeight)fontStyleWeight;
+
+    public static AM.FontStretch ToFontStretch(this SKFontStyleWidth fontStyleWidth) => fontStyleWidth switch {
+        SKFontStyleWidth.UltraCondensed => AM.FontStretch.UltraCondensed,
+        SKFontStyleWidth.ExtraCondensed => AM.FontStretch.ExtraCondensed,
+        SKFontStyleWidth.Condensed => AM.FontStretch.Condensed,
+        SKFontStyleWidth.SemiCondensed => AM.FontStretch.SemiCondensed,
+        SKFontStyleWidth.Normal => AM.FontStretch.Normal,
+        SKFontStyleWidth.SemiExpanded => AM.FontStretch.SemiExpanded,
+        SKFontStyleWidth.Expanded => AM.FontStretch.Expanded,
+        SKFontStyleWidth.ExtraExpanded => AM.FontStretch.ExtraExpanded,
+        SKFontStyleWidth.UltraExpanded => AM.FontStretch.UltraExpanded,
+        _ => AM.FontStretch.Normal
+    };
+    public static SKFontStyleWidth ToSKFontStretch(this AM.FontStretch fontStyleWidth) => fontStyleWidth switch {
+        AM.FontStretch.UltraCondensed => SKFontStyleWidth.UltraCondensed,
+        AM.FontStretch.ExtraCondensed => SKFontStyleWidth.ExtraCondensed,
+        AM.FontStretch.Condensed => SKFontStyleWidth.Condensed,
+        AM.FontStretch.SemiCondensed => SKFontStyleWidth.SemiCondensed,
+        AM.FontStretch.Normal => SKFontStyleWidth.Normal,
+        AM.FontStretch.SemiExpanded => SKFontStyleWidth.SemiExpanded,
+        AM.FontStretch.Expanded => SKFontStyleWidth.Expanded,
+        AM.FontStretch.ExtraExpanded => SKFontStyleWidth.ExtraExpanded,
+        AM.FontStretch.UltraExpanded => SKFontStyleWidth.UltraExpanded,
+        _ => SKFontStyleWidth.Normal
+    };
+
+    public static AM.FontStyle ToFontStyle(this SKFontStyleSlant fontStyleSlant) => fontStyleSlant switch
+    {
+        SKFontStyleSlant.Italic => AM.FontStyle.Italic,
+        SKFontStyleSlant.Oblique => AM.FontStyle.Oblique,
+        _ => AM.FontStyle.Normal
+    };
+    public static SKFontStyleSlant ToSKFontStyle(this AM.FontStyle fontStyleSlant) => fontStyleSlant switch
```

Comment:
Please do not do such refactoring in this PR

Reply:
This is part of AvaloniaAssetLoader implementation